services/provider/server/server.go

```diff
@@ -573,7 +573,8 @@ func decodeAndValidateTicket(ticket string, pubKey *rsa.PublicKey) (*services.On
 // FulfillStorageClaim RPC call to create the StorageClaim CR on
 // provider cluster.
 func (s *OCSProviderServer) FulfillStorageClaim(ctx context.Context, req *pb.FulfillStorageClaimRequest) (*pb.FulfillStorageClaimResponse, error) {
-	return nil, status.Error(codes.Unimplemented, "not implemented")
+	// returning empty response else the old client will always error out
```

Comment:
This Comment does not say anything meaningful. 
- What is the "old client"?
- How would it error out? What would be the behaviour? What would be the end result?